### ISSUE TITLE
device.get_data(): data collected in an advanced timezone is not retrieved

### ISSUE THREAD

**Issue by playertr** [LionelDarras/PyCampbellCR1000#23]
We have our dataloggers set to UTC, but we are doing experiments in UTC-6. 

Consequently, when we call `data = dev.get_data('Radiation', start_date, None)` the automatically assigned stop date of `datetime.datetime.now()` misses the most recent six hours of data because, in an ignoring-time-zone sense, the most recent records have a greater timestamp than the querying computer's local time.

This threw us for a loop because we were expecting an empty `stop_date` argument to retrieve all available records after the start date. A good solution could be either:
- Updating documentation to reveal this behavior
- Searching through all records to find the most recent one (are there costs to this?)
- Requiring an end date parameter

If you have feedback on the solution I can attempt to make a PR. Is this project still maintained? In any case, this project has been greatly appreciated.

**Comment by mesafalcon0** [comment]
I think you can use stop_date= None

If stop_date is None, the get_data command will download up to the most recent data.

**Comment by mesafalcon0** [comment]
I think this might help.
import datetime
stop_date = datetime.datetime.utcnow()

**Comment by playertr** [comment]
Yes, that fix helped in our case -- we ended up doing something similar. Thank you.

More generally, the choice of the default values for `start_date` and `stop_date` in `device.py::get_data_generator()` are potentially confusing, since I would assume that leaving these arguments as None would retrieve ALL data instead of just the data with non-timezoned timestamps between January 1, 1990, and the computer's local time.

https://github.com/LionelDarras/PyCampbellCR1000/blob/9114b2acd47c5c019733f9a2167c557f197ef575/pycampbellcr1000/device.py#L244-L245

In fact, if I understand the rest of that file correctly, modifying those two lines to keep the None argument and pass it to `_collect_data` would cause all data to be collected in the case that `start_date == stop_date == None`. 

The case where only `stop_date == None` would require some different handling due to the structure of the Pakbus API -- as a workaround, potentially we could set `p2` within `_collect_data` to be INT_MAX? 

These are definitely edge cases, but I think a good resolution would be either to document the default behavior or to change the default behavior so it doesn't rely on the seemingly arbitrary default values for `start_date` and `stop_date` in `get_data_generator()`. I'm a slight proponent of just updating the documentation because I suspect this code is embedded in a lot of folks' super-important nichey data applications and that change might break them.
